transports/http/helpers/error_response_helpers.go

```diff
@@ -0,0 +1,48 @@
+package helpers
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/bitcoin-sv/block-headers-service/domains"
+)
+
+// ResponseError is an object that we can return to the client if any error happens
+// on the server
+type ResponseError struct {
+	Code    int    `json:"code"`
+	Message string `json:"message"`
+}
+
+// ErrorResponse is creating an error object to send it to the client
+func ErrorResponse(err error, statusCode int) ResponseError {
+	return parseError(err, statusCode)
+}
+
+// ErrorResponseFromMessage is simplified ErrorResponse when we don't want to create new error
+// in code but just pass the message that will be sent to the client.
+func ErrorResponseFromMessage(errMessage string, statusCode int) ResponseError {
+	return parseError(errors.New(errMessage), statusCode)
+}
+
+// GetCodeFromError returns error code that should be returned to the client
+// in a response based on the error message
+func GetCodeFromError(err error) int {
+	errorMessage := err.Error()
+
+	switch errorMessage {
```

Comment:
I'm not sure about this way of doing this error handling @chris-4chain WDYT?

Reply:
I'm not sure how to handle this differently since we are also throwing errors in repository files. Other way would be to create a custom errors that have both code and message. Could be a better solution

Reply:
After some thinking I don't like the proposed approach. I'd rather do some explicit struct containing data about the error or make it some other way. The way it is proposed generates a huge switch when more and more errors are added. However, I suggest we keep it in mind and do it in another task since this PR already grew quite large